entity-types/infra-mysqlnode/definition.yml

```diff
@@ -1,11 +1,41 @@
 domain: INFRA
 type: MYSQLNODE
+
+synthesis:
+  rules:
+    # https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/0c50ef5a58683a214a509ee15c75279a6334d9b9/receiver/mysqlreceiver/metadata.yaml#L12C9-L12C17
+    - identifier: mysql.instance.endpoint # server.address:server.port
+      name: mysql.instance.endpoint 
+      encodeIdentifierInGUID: true
+      conditions:
+        - attribute: eventType
+          value: Metric
+        - attribute: metricName
+          prefix: mysql.
+        - attribute: instrumentation.provider
+          value: opentelemetry
+        # This filters only metrics coming from mysqlreceiver , given that metrics
+        # could differ between different runtime receivers.
+        - attribute: otel.library.name
+          value: otelcol/mysqlreceiver
+      tags:
+        otel.library.name:
+          entityTagName: instrumentation.name
+        server.address:
+        server.port:
```

Comment:
are these attributes present in the metrics?

Reply:
Hi @zmrfzn, could you point me to where those attributes are being defined for MySQL in OTel? I only see them defined for [redis receiver](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/5a10c4b338edfc559ecbc5fa4d711f6f5716c5f2/receiver/redisreceiver/metadata.yaml#L16).

Reply:
@marcsanmi this is not available in the official collector yet, it was coming from a custom configuration of the receiver. 
I have removed these tags